docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
 # 2. Quick Start Guide
 
 ## 2.1 Overview
-< Describe the general context around this walkthrough/lab >
+
+*Cobalt* is a tool for developers who are interested in reusing or contributing new cloud infrastructure as code patterns in template form. An actualized infrastructure as code pattern in Cobalt is called a *Cobalt Infrastructure Template* or *CIT* (/kɪt/). Cobalt Infrastructure Templates primarily rely on [*Terraform*](https://learn.hashicorp.com/terraform)'s HCL language in order to target a wide array of cloud providers.
+
+You can get pretty creative and build your own custom *CIT*s in order to use and/or contribute to Cobalt but we strongly recommend that you first complete this quick start guide. This guide is centered around our existing [*Azure Hello World CIT*](../infra/templates/az-hello-world/README.md "AZ Hello World - Cobalt Infrastructure Template") and should serve as your first Azure infrastructure deployment. In summary, completing this guide should be your first major step in familiarizing yourself with Cobalt and the *CIT* developer workflow. Happy templating! 😄
+
+> For a more general overview of Cobalt, please visit our main page: [READ ME](https://github.com/microsoft/cobalt/blob/master/README.md "Main Cobalt Read Me")
 
 ## 2.2 Goals and Objectives
-< Outline the goals and takeaways of this walkthrough/lab >
+
+🔲 Prepare local environment for *Cobalt Infrastructure Template* deployments.
+
+🔲 Deploy the [*Azure Hello World CIT*](../infra/templates/az-hello-world/README.md "AZ Hello World - Cobalt Infrastructure Template").
+
+🔲 Walk away with a introductory understanding of the *CIT* developer workflow.
+
+🔲 Feel confident in moving forward to our next recommended section: *[Cobalt Templating from Scratch](https://github.com/microsoft/cobalt/blob/master/docs/3_NEW_TEMPLATE.md).*
 
 ## 2.3 Prerequisites
-< Enumerate any prerequisite walkthroughs that should be completed prior to this, and any technical prerequisites such as developer environment / tools >
 
-## 2.4 Walkthrough
-< Step-by-step instructions for completing this walkthrough >
\ No newline at end of file
+> NOTE: Previous "infrastructure as code" experience is not a prerequisite for completing the quick start guide.
+
+| Azure Prereqs | Description | I | Install Prereqs | Description |
```

Comment:
This table is confusing -- can it be broken into just 2 columns: `Prereqs` and `Description`?

Reply:
I agree, this table is confusing. I'm not sure what the "I" column is intended to do. Also, this seems to be a mixture of "things you need to install first" and "things you need to go configure or enroll first". Maybe would be good to split those two categories of prereqs into different tables?

Reply:
Yeah, I reverted back to a single table. Keeping it simple.